[DE Team][8.15][Serverless] Bulk-update a rule's custom highlighted fields (#5460)
elastic/security-docs: docs/serverless/rules/rules-ui-create.mdx

        false-positive alerts.

    1. **MITRE ATT&CK<sup>TM</sup> threats** (optional): Add relevant [MITRE](https://attack.mitre.org/) framework tactics, techniques, and subtechniques.

>     1. **Custom highlighted fields** (optional): Specify highlighted fields for personalized alert investigation flows. You can choose any fields that are available in the you selected for the rule's data source.

Reviewer: Same comment as above about "personalized."

Author: ```suggestion
    1. **Custom highlighted fields** (optional): Specify highlighted fields for unique alert investigation flows. You can choose any fields that are available in the you selected for the rule's data source.
```

>     1. **Custom highlighted fields** (optional): Specify highlighted fields for personalized alert investigation flows. You can choose any fields that are available in the <DocLink slug="/serverless/security/advanced-settings" section="update-sec-indices">default ((elastic-sec)) indices</DocLink>. There is no limit to the number of fields that you can add. 

Reviewer 2: Same here, on this page `Rule edit > About section` we only show fields that are available in selected index patterns for the rule.

Author: ```suggestion
    1. **Custom highlighted fields** (optional): Specify highlighted fields for personalized alert investigation flows. You can choose any fields that are available in the you selected for the rule's data source.
```
